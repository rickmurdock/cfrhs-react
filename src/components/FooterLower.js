import React from 'react';

const FooterLower = () => {
  return (
    <div className="container-fluid footer-lower">
      <div className="row navbar align-items-center ">
        <div>
        <p className="footer-text-lower my-auto text-center">
          Â© 2019 Central Florida Railway Historical Society, Inc. &nbsp;&nbsp;<i className="fa fa-train fa" aria-hidden="true"></i>&nbsp;&nbsp; Website Designed & Developed by <a className="rick" href="http://rickmurdock.com/" target="_blank" rel='noopener noreferrer'><b><i>Rick Murdock</i></b></a> 
        </p>
        </div>
      </div>
    </div>
  );
};

export default FooterLower;