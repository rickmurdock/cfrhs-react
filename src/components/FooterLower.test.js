import React from 'react';
import renderer from 'react-test-renderer';
import FooterLower from './FooterLower';

test('FooterLower renders correctly', () => {
    const tree = renderer.create(
        <FooterLower />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});