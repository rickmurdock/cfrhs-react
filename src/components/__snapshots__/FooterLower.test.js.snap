// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterLower renders correctly 1`] = `
<div
  className="container-fluid footer-lower"
>
  <div
    className="row navbar align-items-center "
  >
    <div>
      <p
        className="footer-text-lower my-auto"
      >
        © 2017 Central Florida Railway Historical Society, Inc.   
        <i
          aria-hidden="true"
          className="fa fa-train fa"
        />
           Website Designed & Developed by 
        <a
          className="rick"
          href="http://rickmurdock.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <b>
            <i>
              Rick Murdock
            </i>
          </b>
        </a>
      </p>
    </div>
  </div>
</div>
`;
